lib/convection/model/cloudfile.rb

```diff
@@ -61,8 +63,22 @@ def initialize(cloudfile)
         @stacks = {}
         @deck = []
         @stack_groups = {}
+        @thread_count = thread_count || 2
```

Comment:
No because the thread_count is the value specified in the cloud file so i only want it to be 2 if its not specified.